### DISCUSSION TITLE
BOM Item reference field length once again

### DISCUSSION THREAD

**Discussion by SergeoLacruz** [inventree/InvenTree#4015] [Q&A] [answered]
Hi,
I know there was already a discussion about this and I do not want to start another one regarding performance and response time.
We just need more characters  here. 
So I gave it a try and did three changes as shown here:

https://github.com/SergeoLacruz/InvenTree/tree/long_reference

I also changed the field length in the existing database with the sql console. Just an experiment...
In my setup it works nicely with a BOM  with 225 components and about 2500 parts in total. The longest reference field is 1200 characters. I still have very good response times.

Questions: Are there more changes required? Unfortunately I do not have an overview and I am sure I missed something. 
Thanks in advance,

Michael

**Comment by SchrodingersGat** [comment, marked as answer]
You should not edit the old historical migration files. These should not be changed except for under very special circumstances. 

Instead, only edit the source file (part.models.py) and then run `invoke migrate`. A new migration file will be created.

**Reply by SergeoLacruz** [reply]
This was a quick change. Works great. Thanks for the help.
